api/types/client.go

```diff
@@ -292,6 +292,7 @@ type ServiceListOptions struct {
 // TaskListOptions holds parameters to list  tasks with.
 type TaskListOptions struct {
 	Filter filters.Args
+	All    bool
```

Comment:
This shouldn't require such a literal API change. Shouldn't an empty filter send all?